docs/concepts/bot-filtering.md

```
@@ -0,0 +1,46 @@
+    return ua.includes('my-custom-bot');
+  },
+});
+```
+
+When `shouldIgnoreUserAgent` is not provided, `isbot` is used as the default.
+
+## When Bot Check Runs
+
+The `User-Agent` header is checked at the start of `handleEventRequest()`, before any parsing or enrichment. If the check passes (returning `true`), the handler exits immediately. If `debug` mode is enabled, a log message is emitted:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)